src/components/02-components/past-experiences-page/past-experiences-page.tsx

```diff
@@ -0,0 +1,106 @@
+import {
```

Comment:
The table should be an individual component, don't include the pending tasks card as part of it.